common/log/zap_logger.go

```diff
@@ -54,8 +55,12 @@ var _ Logger = (*zapLogger)(nil)
 
 // NewTestLogger returns a logger at debug level and log into STDERR
 func NewTestLogger() *zapLogger {
+	lvl := "debug"
+	if override, ok := os.LookupEnv("TEMPORAL_TEST_LOG_LEVEL"); ok {
```

Comment:
I don't like using envs from code in many random places. It is like a global var. I have plans to clean it up and I would like not to add new ones. Here, I would just change it to "info" or even "warning".
I don't have strong idea on how to clean it up though. So if you just want to leave it as is, I am also ok with.

Reply:
I don't mind dropping this. I did it for my own purposes in debugging some test failures and it seemed broadly useful. But I can just drop this commit from the PR.